src/tasks.ts

```diff
@@ -18,9 +18,11 @@ interface MesonTaskDefinition extends vscode.TaskDefinition {
 function createTestTask(t: Test, buildDir: string, isBenchmark: boolean) {
   const project = t.suite[0].split(":")[0]
   const name = `${project}:${t.name}`;
-  const mode = isBenchmark ? "benchmark" : 'test'
-  const benchmarkArgs = isBenchmark ? ["--benchmark", "--verbose"] : [];
-  const args = ["test", ...benchmarkArgs].concat(name);
+  const mode = isBenchmark ? "benchmark" : "test";
+  const verboseArgs = (isBenchmark || extensionConfiguration("verboseTest")) ? ["--verbose"] : [];
```

Comment:
Something I've been wondering before, why are benchmark verbose by default? Maybe we should remove this different now that we have an option?

Reply:
That was from me in #48. Didn't see the point of running a benchmark and not looking at the output :)